Podcast/BookmarkTableViewCell.swift

```diff
@@ -148,12 +148,9 @@ class BookmarkTableViewCell: UITableViewCell {
         recommendedButton.setTitle(numberOfRecommendations, for: .normal)
         recommendedButton.setTitle(numberOfRecommendations, for: .selected)
         recommendedButton.isSelected = episode.isRecommended
-        
-        episodeImage.image = #imageLiteral(resourceName: "nullSeries")
+    
+        episodeImage.setImageAsynchronouslyWithDefaultImage(url: episode.smallArtworkImageURL)
```

Comment:
Who wrote ImageView.swift? Not sure why we don't just use Haneke (or a better library that isn't outdated af) directly

Reply:
It's a nice abstraction to have a wrapper class so not every file has to import Haneke